# Goal definition
float32 target_distance

---

# Result definition
string confirmation

---

# Feedback definition
float32 current_distance
